Adafruit_MCP23017.cpp

```diff
@@ -137,6 +137,76 @@ void Adafruit_MCP23017::begin(uint8_t addr, TwoWire *theWire) {
  */
 void Adafruit_MCP23017::begin(TwoWire *theWire) { begin(0, theWire); }
 
+/**
+ * Changes the adress to which MCP23017 is talked to.
+ * Initializing all ports as input is optional.
+ * This enables the option to control multiple MCP23017 with one instance.
+ * Both adress syntax (i2x) = 0x20 or only the configurable part is supported.
+ */
+void Adafruit_MCP23017::changeAddress(uint8_t addr, bool initAsInput) {
```

Comment:
Both this and `changeAddress` on line 171 are defined with a uint8_t and bool as arguments, and it looks like that's the cause of the failure

Reply:
I'm sorry not sure how this happened...

Reply:
No worries. Thanks for fixing it.